app/views/documents/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :page_title, "#{@client.preferred_name} ##{@client.id} documents" %>
+
+<% content_for :card do %>
+  <%= render "clients/client_header" %>
+
+  <%= render "clients/navigation" %>
+
+  <table class="data-table document-table spacing-above-25">
+    <thead>
+    <tr>
+      <th class="document-column__doc-type"><%=t(".document_type") %></th>
+      <th class="document-column__link"><%=t(".file_name") %></th>
+      <th class="document-column__upload-date"><%=t(".upload_date") %></th>
+    </tr>
+    </thead>
+    <tbody>
+    <% if @documents.present? %>
+      <% @documents.each do |document| %>
+        <tr id="document-<%= document.id %>">
+          <td><%= document.document_type %></td>
+          <td>
+            <%= link_to document_path(id: document.id), target: "_blank" do %>
```

Comment:
Would be nice to toss `rel="noopener noreferrer"` in there, too, to avoid an arguably-minor confusion/security-issue vector https://sites.google.com/site/bughunteruniversity/nonvuln/phishing-with-window-opener